search-sentiments-map/test/country-trends-test.js

```diff
@@ -146,42 +112,62 @@ describe('Country Trends', function() {
 
     it('should retrieve the most recent array of US and UA country trends from the datastore beyond the time it takes to retrieve search results', 
         async function() {
+      // Stub calls to the datastore.
+      sinon.stub(Datastore.prototype, 'runQuery').callsFake(() => {
+        return [mockData];
+      });    
```

Comment:
So I think you would want to declare the stub and save it as a variable in the before each, and you could just set the return here.

So for e.g. in the beforeEach have `runQueryStub = sinon.stub(Datastore.prototype, 'runQuery');`
And then within each "it" block `runQueryStub.callsFake(....)`.

On a separate note. Since you aren't actually using the parameters. You may be able to just do `runQueryStub.returns([mockData])`. Also since runQuery should be asynchronous, it may actually need to be `runQueryStub.resolves([mockData])`.